docs/resources/object_bucket.md

```diff
@@ -40,6 +119,27 @@ The `CORS` object supports the following:
 * `expose_headers` (Optional) Specifies expose header in the response.
 * `max_age_seconds` (Optional) Specifies time in seconds that browser can cache the response for a preflight request.
 
+The `lifecycle_rule` (Optional) object supports the following:
+
+* `id` - (Optional) Unique identifier for the rule. Must be less than or equal to 255 characters in length.
+* `prefix` - (Optional) Object key prefix identifying one or more objects to which the rule applies.
+* `tags` - (Optional) Specifies object tags key and value.
+* `enabled` - (Required) Specifies lifecycle rule status.
+* `abort_incomplete_multipart_upload_days` (Optional) Specifies the number of days after initiating a multipart upload when the multipart upload must be completed.
+* `expiration` - (Optional) Specifies a period in the object's expire (documented below).
+* `transition` - (Optional) Specifies a period in the object's transitions (documented below).
+
+At least one of `abort_incomplete_multipart_upload_days`, `expiration`, `transition` must be specified.
+
+The `expiration` object supports the following
+
+* `days` (Optional) Specifies the number of days after object creation when the specific rule action takes effect.
+
+The `transition` object supports the following
+
+* `days` (Optional) Specifies the number of days after object creation when the specific rule action takes effect.
+* `storage_class` (Required) Specifies the Scaleway storage class `STANDARD`, `GLACIER`, `ONEZONE_IA`  to which you want the object to transition.
```

Comment:
Could you add a link to https://www.scaleway.com/en/docs/storage/object/concepts/#storage-class